types/handler.go

```diff
@@ -153,5 +153,11 @@ func (t Terminator) AnteHandle(ctx Context, _ Tx, _ bool, _ AnteHandler) (Contex
 
 // Simply return provided txDeps and nil error
 func (t Terminator) AnteDeps(txDeps []sdkacltypes.AccessOperation, _ Tx, _ AnteDepGenerator) ([]sdkacltypes.AccessOperation, error) {
-	return txDeps, nil
+	// TODO:: use the commit access helper - blocked by circular dep. Should move it to the type/accesscontrol instead
+	commit := sdkacltypes.AccessOperation{
+		ResourceType: sdkacltypes.ResourceType_ANY,
```

Comment:
can you help me understand why we need this if ante handler will eventually be followed by the actual msg's dependency list which already ends with a COMMIT?

Reply:
actually, yeah that's a good point. Let me remove this and validate the access ops another way - the validate access ops call always assert that there's a COMMIT at the end, this breaks down when we're just validating the ante handlers